core/model/src/main/java/org/eclipse/rdf4j/model/util/Configurations.java

```diff
@@ -21,21 +21,27 @@
 import org.eclipse.rdf4j.model.Model;
 import org.eclipse.rdf4j.model.Resource;
 import org.eclipse.rdf4j.model.Value;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Utility functions for working with RDF4J Models representing configuration settings.
  *
  * @author Jeen Broekstra
- *
  * @implNote intended for internal use only.
  */
 @InternalUseOnly
 public class Configurations {
 
+	private static final boolean USE_CONFIG = "true"
+			.equalsIgnoreCase(System.getProperty("org.eclipse.rdf4j.model.vocabulary.experimental.enableConfig"));
```

Comment:
I've introduced a system property to control if we should use the new CONFIG stuff.
 - "true" -> use CONFIG when adding data to the models and ignore legacy vocabulary when reading from a model
 - anything else -> use the legacy vocabulary when adding data to the models, but support both the lagacy and CONFIG vocabularies when reading